docs/environment_variables.rst

```diff
@@ -0,0 +1,34 @@
+.. currentmodule:: astropy
+
+.. _environment_variables:
+
+*********************
+Environment variables
+*********************
+
+
+Runtime
+-------
```

Comment:
What do you need this for?

Reply:
The idea is to leave room for other sections like "build-time" and "tests and CI", were other environment variables play a role.

Reply:
Why do you have to worry about that now?

Reply:
Depends if we want this PR to be minimal or not.
Given that it emerged as an extra requirement from another PR, I'm now worried that scope creep could eventually kill the real target (#17640), so I'm going with minimalism for now.

Reply:
The minimalist approach would be to not introduce sub-sections if there's only one of them.

Reply:
Indeed, that's why I removed it. Isn't this thread marked as outdated ?

Reply:
It is outdated, I didn't notice.